provisioning/transport/mqtt/src/Microsoft.Azure.Devices.Provisioning.Transport.Mqtt.csproj

```diff
@@ -9,6 +9,7 @@
     <OutputPath>.\bin\$(Configuration)</OutputPath>
     <DocumentationFile>$(OutputPath)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
     <RootDir>$(MSBuildProjectDirectory)\..\..\..\..</RootDir>
+    <NoWarn>CA2007</NoWarn>
```

Comment:
remove this change

Reply:
We cannot have ConfigureAwait with DotNetty. See other comment.

---
In reply to: [217796093](https://github.com/Azure/azure-iot-sdk-csharp/pull/619#discussion_r217796093) [](ancestors = 217796093)

Reply:
See above: there is always an implicit configuration of the `await`ed operation. (Default is `true`.)
Default FXCop rules are making the `ConfigureAwait` usage mandatory: your semantic is identical to `ConfigureAwait(true)`. I don't agree with that either since the control has already been returned to the dotNetty thread when the first `await` is encountered.